ci/check-params-env.sh

```diff
@@ -225,6 +225,21 @@ function check_image_variable_matches_name_and_commitref() {
             expected_commitref="release-2023b"
             expected_build_name="codeserver-ubi9-python-3.9-amd64"
             ;;
+        odh-rocm-minimal-notebook-image-n)
+            expected_name="odh-notebook-jupyter-minimal-ubi9-python-3.9"
+            expected_commitref="2024a"
+            expected_build_name="rocm-jupyter-minimal-ubi9-python-3.9-amd64"
+            ;;
+        odh-rocm-pytorch-notebook-image-n)
+            expected_name="odh-notebook-jupyter-rocm-pytorch-ubi9-python-3.9"
+            expected_commitref="2024a"
+            expected_build_name="rocm-jupyter-pytorch-ubi9-python-3.9-amd64"
+            ;;
+        odh-rocm-tensorflow-notebook-image-n)
+            expected_name="odh-notebook-jupyter-rocm-tensorflow-ubi9-python-3.9"
+            expected_commitref="2024a"
```

Comment:
As we plan to include these images in `2024b` and we don't have any image build for `2024a` anyway, keep the following there for now so this CI pass and we'll update later:
```
#expected_commitref="2024b"
expected_commitref="main"
```